src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -734,7 +737,12 @@ class UnifiedShareModal extends React.Component<Props, State> {
                     className="unified-share-modal"
                     isOpen={isConfirmModalOpen ? false : isOpen}
                     onRequestClose={submitting ? undefined : onRequestClose}
-                    title={<span>{this.renderModalTitle()}</span>}
+                    title={
+                        <span>
+                            {this.renderModalTitle()}
+                            {classificationBtn}
+                        </span>
+                    }
```

Comment:
Seeing following notifications as making commit:
:heavy_multiplication_x:   header must not be longer than 72 characters [header-max-length]
:heavy_multiplication_x:   message may not be empty [subject-empty]
:heavy_multiplication_x:   type may not be empty [type-empty]